cocos2dx/actions/CCActionEase.h

```diff
@@ -407,7 +407,7 @@ class CC_DLL CCEaseElasticInOut : public CCEaseElastic
 class CC_DLL CCEaseBounce : public CCActionEase
 {
 public:
-    float bounceTime(float time);
+    static float bounceTime(float time);
```

Comment:
When change a member function to static function. The original object can't access the member function any more. So I think it will break back compatability.